### PR TITLE
New optional argument 'doNotMoveCursor' for iTerm2 images.

### PR DESCRIPTION
**Pull request by klamonte**
For #1424

New optional argument 'doNotMoveCursor' for iTerm2 images.  This permits iTerm2 images to be drawn anywhere on screen without scrolling the cursor, including the bottom row.

Also included is a check in fcwrap.rs to_range_set(), without which was causing a panic at runtime due to subtraction from unsigned leading to overflow.

(Sidebar: I had to switch to Rust nightly to compile git head, due to the sysinfo dependency not compiling with Rust stable.  Is that intended behavior?)

### CODE REVIEW

**Comment by wez** [comment]
re: rust version, stable Rust 1.57 builds things just fine.  I suspect that you're using a distribution-provided version of Rust that hasn't caught up to the latest stable Rust.  I recommend using `rustup` to manage the Rust version!

Would you mind sharing which version of stable rust you were using? It sounds like we should update the `ci/check-rust-version.sh` to reflect the minimum version requirement from sysinfo.

**Comment by klamonte** [comment]
My stable version currently is:

stable-x86_64-unknown-linux-gnu unchanged - rustc 1.57.0 (f1edd0429 2021-11-29)

But it had been quite a while since I did an upgrade before yesterday, so can't know for sure if the failed compile was at this version.  I immediately switched to nightly and continued on.

File: wezterm-font/src/fcwrap.rs [hunk 1]
```diff
@@ -125,7 +125,7 @@ impl<'a> CharSetRef<'a> {
                 for j in 0..32 {
                     if mask & (1 << j) != 0 {
                         let new_code_point = base_code_point + (j + i * 32) as u32;
-                        if new_code_point - 1 > code_point {
+                        if new_code_point > 0 && new_code_point - 1 > code_point {
```
Comment (wez) [inline wezterm-font/src/fcwrap.rs:128]: ah, this looks like it doesn't belong in this PR!

Comment (wez) [reply wezterm-font/src/fcwrap.rs:128]: I'll push this in as a separate commit (bd09b568ed31401a5b525aad40c5d3817e55e486); hopefully this one will rebase out without issue

**Comment by wez** [comment]
Thanks!
